src/host/buildvm.c

```diff
@@ -110,7 +111,7 @@ static const char *sym_decorate(BuildCtx *ctx,
   if (p) {
 #if LJ_TARGET_X86ORX64
     if (!LJ_64 && (ctx->mode == BUILD_coffasm || ctx->mode == BUILD_peobj))
-      name[0] = '@';
+      name[0] = isupper(name[1]) ? '_' : '@';
```

Comment:
I'd rather check for `name[1] == 'R'` and add a comment that this is only for `_RtlUnwind@16`.